service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -99,7 +115,19 @@ class ScaladslKafkaApiSpec
 
   override def afterAll(): Unit = {
     application.application.stop().futureValue
-    kafkaServer.stop()
+
+    kafkaServer.foreach { process =>
+      try {
+        process.destroy()
+        if (!process.waitFor(10, TimeUnit.SECONDS)) {
+          process.destroyForcibly()
+          process.waitFor(10, TimeUnit.SECONDS)
+        }
+      } catch {
+        case NonFatal(_) => ()
+      }
+    }
+    kafkaServer = None
```

Comment:
If this works, can we extract the duplication here and the javadsl and reuse?

Reply:
build-tool-support-2?

Yeah, we should.  And it should be shared with build-tool-support.  But I'm pushing that to another day.

Reply:
> build-tool-support-2?

`kafka-build-support`?

Reply:
Pushed to https://github.com/lagom/lagom/issues/2041.